challenges/treeIntersection/README.md

```diff
@@ -0,0 +1,34 @@
+## Tree Intersection
+
+
+## code challenge 32
+
+
+
+### challenge
+
+- Find common values in two binary trees.
+
+- Write a function called tree-intersection that takes two binary tree parameters
+
+- Without utilizing any of the built-in library methods available to your language, return a set of values found in both trees.
+
+### Approach
+
+- traverse the first tree inorder and store the value in an array
+
+- traverse the second tree inorder and store the value in an array
+
+- find intersection of both arrays
+
+
+
+
+
+### attribiution to:
+
+
+- *Carly Dekock*
+- *Jason D*
+- *Jason Q*
+- *Nick M*
```

Comment:
Every looks good so far Seid! Once you have a chance to come back and finish I will regrade for you!